src/openssl.cpp

```diff
@@ -78,7 +78,8 @@ ssl::context::method ssl_version(int const v)
 		case settings_pack::ssl23: return ssl::context::sslv23;
 		case settings_pack::tls11: return ssl::context::tlsv11;
 		case settings_pack::tls12: return ssl::context::tlsv12;
-		default: return ssl::context::tlsv12;
+		case settings_pack::tls13: return ssl::context::tlsv13;
```

Comment:
Maybe guard against older boost versions? I'm fine if not but seems recent enough.